scss/_main-screen.scss

```
@@ -0,0 +1,100 @@
+.first-screen {
+  background: url(../img/HeaderBackground.jpg) center center no-repeat;
+  background-size: cover;
+  height: 862px;
+  color: rgba(255, 255, 255, 1);
+  & * {
+    display: flex;
+    justify-content: center;
+  }
+}
+
+h1 {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0em;
+}
+
+.h1__align {
+  padding-top: 295px;
+  padding-bottom: 8px;
+}
+
+.subheader {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0.31em;
+}
+
+.input__align {
+  display: flex;
+  margin-top: 48px;
+  margin-bottom: 211px;
+}
+
+input {
+  height: 60px;
+  width: 647px;
```

лучше так, чтоб не вылезло на адаптиве
```suggestion
  width: 100%;
  max-width: 647px;
```